### PR TITLE
[Admin] Create a page for HQ's missing receipts

### PR DESCRIPTION
**Pull request by sampoder**
https://hackclub.slack.com/files/USNPNJXNX/F06SVJ42LT0/screenshot_2024-04-05_at_12.25.07___pm.png

### CODE REVIEW

File: app/controllers/admin_controller.rb [hunk 1]
```diff
@@ -1155,6 +1155,14 @@ def column_statements
     render layout: "admin"
   end
 
+  def hq_receipts
+    @page = params[:page] || 1
+    @per = params[:per] || 20
+    @users = User.where(id: Event.where(category: 7).or(Event.where(omit_stats: true)).flat_map(&:users).map(&:id)).page(@page).per(@per).order(created_at: :desc)
```
Comment (garyhtou) [inline app/controllers/admin_controller.rb]:
```suggestion
    @users = User.where(id: Event.hack_club_hq.or(Event.omitted).includes(:users).flat_map(&:users).map(&:id)).page(@page).per(@per).order(created_at: :desc)
```

**Review by garyhtou — COMMENTED** [review]
thank you! I'd also like a way to see a list of their cards. This way we can freeze them if necessary

**Comment by sampoder** [comment]
Added that!

File: app/views/admin/hq_receipts.html.erb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            <details class="mb0" style="background: #ff8c3830;">
+              <summary>
+                <%= user.transactions_missing_receipt.count %>
+                <%= "Transaction".pluralize(user.transactions_missing_receipt.count) %> Missing Receipts
+              </summary>
+              <ul>
+                <% user.transactions_missing_receipt.each do | txn | %>
+                  <li>
+                    <%= link_to txn.memo, txn %>
+                    (<%= link_to txn.stripe_card.hidden_card_number_with_last_four, txn.stripe_card %>)
```
Comment (garyhtou) [inline app/views/admin/hq_receipts.html.erb]:
```suggestion
                    (<%= link_to "x#{txn.stripe_card.last4}", txn.stripe_card %>)
```

**Review by garyhtou — DISMISSED** [review]
thank you!

**Comment by garyhtou** [comment]
If you'd like feel free to follow up with another pr to order users by number of missing receipts desc

**Pull request merged**